### ISSUE TITLE
ValueError: Expected x_max for bbox to be in the range [0.0, 1.0], got 1.0025.

### ISSUE THREAD

**Issue by atikapane** [albumentations-team/albumentations#922]
## 🐛 Bug

I tried to transform images using RandomResizedCrop with its bounding box (yolo format). But, there are images that when the program tried to transform the bounding box, I got error saying that the bounding box value (after transformation) is not in the range of 0-1. I already checked the original bounding box, and there is nothing wrong with the original bounding box value.

This is my pipeline
```
transform = A.Compose([
    A.RandomResizedCrop(608, 608),
], bbox_params = A.BboxParams(format='yolo'))
```

This is the code to place the bounding box to an array and transform it
```
    bboxes = []
    for dt in data:
        ann = []

        # Split string to float
        c, x, y, w, h = map(float, dt.split(' '))
        ann.append(x)
        ann.append(y)
        ann.append(w)
        ann.append(h)
        ann.append(c)

        bboxes.append(ann)

    transformed = transform(image=img, bboxes=bboxes)
```
And this is the error
```
  File "C:\Users\Atika Pane\AppData\Roaming\Python\Python38\site-packages\albumentations\core\composition.py", line 180, in __call__
    p.preprocess(data)
  File "C:\Users\Atika Pane\AppData\Roaming\Python\Python38\site-packages\albumentations\core\utils.py", line 62, in preprocess
    data[data_name] = self.check_and_convert(data[data_name], rows, cols, direction="to")
  File "C:\Users\Atika Pane\AppData\Roaming\Python\Python38\site-packages\albumentations\core\utils.py", line 70, in check_and_convert
    return self.convert_to_albumentations(data, rows, cols)
  File "C:\Users\Atika Pane\AppData\Roaming\Python\Python38\site-packages\albumentations\augmentations\bbox_utils.py", line 51, in convert_to_albumentations
    return convert_bboxes_to_albumentations(data, self.params.format, rows, cols, check_validity=True)
  File "C:\Users\Atika Pane\AppData\Roaming\Python\Python38\site-packages\albumentations\augmentations\bbox_utils.py", line 302, in convert_bboxes_to_albumentations
    return [convert_bbox_to_albumentations(bbox, source_format, rows, cols, check_validity) for bbox in bboxes]
  File "C:\Users\Atika Pane\AppData\Roaming\Python\Python38\site-packages\albumentations\augmentations\bbox_utils.py", line 302, in <listcomp>
    return [convert_bbox_to_albumentations(bbox, source_format, rows, cols, check_validity) for bbox in bboxes]
  File "C:\Users\Atika Pane\AppData\Roaming\Python\Python38\site-packages\albumentations\augmentations\bbox_utils.py", line 251, in convert_bbox_to_albumentations
    check_bbox(bbox)
  File "C:\Users\Atika Pane\AppData\Roaming\Python\Python38\site-packages\albumentations\augmentations\bbox_utils.py", line 327, in check_bbox
    raise ValueError(
ValueError: Expected x_max for bbox (0.9675, 0.19469026548672566, 1.0025, 0.3274336283185841, 0.0) to be in the range [0.0, 1.0], got 1.0025.
```

**Comment by ternaus** [comment]
Valid values lie in `[0, 1)` and not `[0, 1]`

**Comment by atikapane** [comment]
But why is it I got error like this when I'm trying to transform the image?
`ValueError: Expected x_max for bbox (0.9675, 0.19469026548672566, 1.0025, 0.3274336283185841, 0.0) to be in the range [0.0, 1.0], got 1.0025.`

This is the original bounding box
```
0 0.537500 0.227876 0.130000 0.323009
0 0.982500 0.263274 0.035000 0.137168
0 0.351250 0.471239 0.112500 0.172566
```

**Comment by ternaus** [comment]
Am I right that for the second box: 

`x_min = 0.9825` 
`x_max = 0.9825 + 0.035 = 1.0175` ?

**Comment by Dipet** [comment]
Hi, looks like known issue. These issues looks the same: 
- #903
- #862 
- #883 
- #848 
- #679 

I will try to fix this problem today.

**Comment by Dipet** [comment]
Should be fixed by #924
